### PR TITLE
new hook description and changed related files

### PR DESCRIPTION
**Pull request by fiedsch**
Added a new hook description for `sqlGetFromDca` and modified the
related files and the table of contents file

### CODE REVIEW

File: api/hooks/sqlGetFromDca.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# sqlGetFromDca
+
+The `sqlGetFromDca` hook is triggered when sql definitions in DCA files are evaluated. It passes
+the generated SQL definition and expects the same as return value.
+
+> **Tag** Available from version 3.5.0-beta1.
```
Comment (aschempp) [inline api/hooks/sqlGetFromDca.md]: The hook was added in Contao 3.2 (see https://github.com/contao/core/commit/869812c48c0a90e62515d64ecedcfe5f3a100da9)

File: api/hooks/sqlGetFromDca.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+
+## Parameters
+
+1. *array* `$arrReturn`
```
Comment (aschempp) [inline api/hooks/sqlGetFromDca.md]: I would prefer `$arrSql`, what do you think?

Comment (fiedsch) [inline api/hooks/sqlGetFromDca.md]: Sounds reasonable. I'll change `sqlGetFromFile.md` and `sqlCompileCommands.md` too as this is mostly the same description

Comment (aschempp) [inline api/hooks/sqlGetFromDca.md]: Thanks. Please let me know when the branch is updated.

**Comment by fiedsch** [comment]
updated the PR to include the suggested changes

**Pull request merged**